client/src/pages/Chat.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useRef, useState } from 'react';
 import { useNavigate, useSearchParams, createSearchParams } from 'react-router-dom';
+import useDetectKeyboardOpen from "use-detect-keyboard-open";
```

Comment:
Doesn't the onFocus event on the input box can achieve the same result but without the external, not fully released library?

Reply:
No, because an input field can remain focused even when the virtual keyboard is closed, so it is not an indication for the wanted screen size